CHANGELOG.md

```diff
@@ -5,7 +5,23 @@ All notable changes to htmltools for Python will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [UNRELEASED]
+## [Unreleased] YYYY-MM-DD
+
+### Breaking changes
+
+* `HTML` no longer inherits from `str`. It now inherits from `collections.UserString`. This was done to avoid confusion between `str` and `HTML` objects. (#86)
+
+* `Tag` attributes now error when given an `HTML()` object. To fix, use `str()` to get a string (e.g., `div(foo=str(HTML("bar")))`. (#86)
+
+* `Tag` and `TagList`'s method `.get_html_string()` now both return `str` instead of `HTML`. (#86)
+
+* `TagAttrDict` no longer silently supports `HTML` values for attrs. Only `str` values are supported. (#86)
```

Comment:
We should still support `HTML` values for tag attributes. For example, this is the existing behavior, and we want to keep it:

```
>>> ui.div(x="1<2&")
<div x="1&lt;2&amp;"></div>

>>> ui.div(x=ui.HTML("1<2&"))
<div x="1<2&"></div>
```